src/System.Management.Automation/engine/runtime/CompiledScriptBlock.cs

```diff
@@ -1520,11 +1520,7 @@ private static bool WriteScriptBlockToLog(ScriptBlock scriptBlock, int segment,
 
```

Comment:
@TravisEz13, your last commit had 2 failures in `PowerShell-CI-linux`
Basic SysLog tests on Linux.Verifies scriptblock logging
```powershell
Expected regular expression 'Creating Scriptblock text \(1 of 1\):#012\$pid?& \(\[scriptblock\]::create\("Write\-Verbose 'testheader123' ;Write\-verbose 'after'"\)\)(#012)*ScriptBlock ID: [0-9a-z\-]*#012Path:.*' to match 'Creating Scriptblock text (1 of 1):#012$pid?& ([scriptblock]::create("Write-Verbose 'testheader123' ;Write-verbose 'after'"))?#012#012ScriptBlock ID: 75103d64-1b5a-4c80-980b-66f3a0872bc4#012Path: /tmp/c9508df5-4a20-418d-a42a-a04810ee373d/test01.ps1', but it did not match.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Host/Logging.Tests.ps1: line 214
214:         $createdEvents[1].Message | Should -Match ($scriptBlockCreatedRegExTemplate -f (Get-RegEx -SimpleMatch $Script.Replace([System.Environment]::NewLine,"`u{23CE}")))
```
Basic SysLog tests on Linux.Verifies scriptblock logging with null character
```powershell
Expected regular expression 'Creating Scriptblock text \(1 of 1\):#012\$pid?& \(\[scriptblock\]::create\("Write\-Verbose 'testheader123\$\(\[char\]0x0000\)' ;Write\-verbose 'after'"\)\)(#012)*ScriptBlock ID: [0-9a-z\-]*#012Path:.*' to match 'Creating Scriptblock text (1 of 1):#012$pid?& ([scriptblock]::create("Write-Verbose 'testheader123$([char]0x0000)' ;Write-verbose 'after'"))?#012#012ScriptBlock ID: abdbfad9-56d8-4149-b909-d790e3fcaed1#012Path: /tmp/c9508df5-4a20-418d-a42a-a04810ee373d/test01.ps1', but it did not match.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Host/Logging.Tests.ps1: line 243
243:         $createdEvents[1].Message | Should -Match ($scriptBlockCreatedRegExTemplate -f (Get-RegEx -SimpleMatch $Script.Replace([System.Environment]::NewLine,"`u{23CE}")))
```